src/Containers/Dashboard/Dashboard.js

```diff
@@ -545,7 +547,8 @@ class Dashboard extends Component {
   }
 
   async fetchBurnPool() {
-    const burnBal = await nrtManager.burnPoolBalance();
+    // const burnBal = await nrtManager.burnPoolBalance();
```

Comment:
This was already correct. Burn pool is inside NRT manager contract, it is not on private key wallet. This needs to be reverted.